x-pack/plugins/security_solution/server/endpoint/services/workflow_insights/helpers.test.ts

```diff
@@ -193,4 +253,13 @@ describe('helpers', () => {
       });
     });
   });
+
+  describe('generateInsightId', () => {
+    it('should generate the correct hashed id', () => {
+      const insight = getDefaultInsight();
+      const result = generateInsightId(insight);
+      const expectedHash = '6b1a7a9625decbf899db4fbf78105a0eff9ef98e3f2dadc2781d59996b55445e';
+      expect(result).toBe(expectedHash);
+    });
+  });
```

Comment:
We’re testing the `crypto` package, not our code here ;)

Reply:
this catches if the fields or hashing algo are changed.